flow/__main__.py

```diff
@@ -46,6 +49,43 @@ def main_init(args):
         )
 
 
+def main_template(args):
+    """Print out general information about template subcommand."""
+    print("flow template does nothing. Use of the available subcommands [create,].")
+
+
+def main_template_create(args):
+    """Create custom template based on a specified or detected environment."""
+    if args.name is None:
+        env = environment.get_environment()
+        if env is environment.StandardEnvironment:
+            extend_template = "base_script.sh"
```

Comment:
Instead of having a hack right here, should we define the template for the `StandardEnvironment` to be `base_script.sh`?

As a separate question, does it make any sense to allow template creation if the specified/detected environment has no scheduler? With #533, is it even possible to generate a script on environments without a scheduler? (Reasons: we removed the `script` command, `submit` can't do anything without a scheduler, and `submit --pretend` will also do nothing after #533.)

Reply:
Maaaaybe. I'm not sure. I'm inclined to warn the user if `env.scheduler_type is None` with a message like:
> `"No scheduler is defined for this environment. A template can be generated, but a scheduler will be required for submission."`

This special case for `StandardEnvironment` doesn't do anything as far as I can tell. Currently the code behaves as:
```python
>>> import flow
>>> flow.environment.StandardEnvironment.template
'base_script.sh`
```

Reply:
Seems like it is not, I didn't see it defined in `StandardEnvironment` (it is set in `ComputeEnvironment`), so I will remove it.